meeting-notes/2020-03-04.md

```diff
@@ -0,0 +1,87 @@
+    - Unclear who owns repo
+        - Chris thinks anyone should be able to contribute except on some files
+        - Carolyn will put up PR
+- Working document for CNAB Registries
+    - Chris: we have a structure in place - anyone who wants to contribute is invited to, Chris will start to document how `cnab-to-oci` works right now
+    - Chris: do we have a process of ratifying other specs?
+    - Matt: we have the same process as for the core spec, vote has to be unanimous.
+    - Chris: it may be worth having external people review the security document (Justin Cormack)
+- Jacob's work on [#285](https://github.com/cnabio/cnab-spec/issues/285)
+    - Please take a look at these!
+    - Show complexity of making bundle internals visible even if you know the target
+    - [Spec PR #337](https://github.com/cnabio/cnab-spec/issues/337)
+        - Declarative credential scoping
+        - Builds on extensions work done by Simon and Silvin
+        - Can add this as non-core spec
+        - Carolyn: Is idea that bundle says these are the credentials the bundle needs and that the expectation is that the runtime creates user/service config with these?
+            - Jacob: Idea is to pass admin credentials and have them down privileged
+                - Unfortunately this isn't practically easy to do
+        - Carolyn: Where would the code that does this live, CNAB Go?
+            - Matt: This would be non-normative so could be implemented in an external tool
+            - Jacob: For CNAB Go, we could have an interface for multiple credential providers and that spits out credential sets
+            - Carolyn: Wondering how portable bundles like this would be
+                - Jacob: Goal is to have something portable and avoid having bundles only be able to be installed by one tool
+                - Carolyn: Tool that understands this extension has better security, naive tool still works
+    - [Spec PR #338](https://github.com/cnabio/cnab-spec/issues/338)
+        - Declarative installers for Kubernetes
+    - Radu: Feels like using OCI manifests to share Kubernetes YAML has suddenly come up
+        - Not sure that CNAB is the right place to solve this
+        - Jacob: Don't have this use case but wanted to show that it could be done
+        - Matt: Need to work out if this is an issue impacting enough users
+- [CNAB Go #188](https://github.com/cnabio/cnab-go/pull/188)
+    - Validate schema offline
+    - Schemas currently live online (cnab.io/), we currently fetch the schema to do validation
+    - This doesn't work for airgapped scenarios
+    - Docker has done this in docker/cli for Compose
+    - Radu doesn't want to copy paste the schema in a Go file
+    - Bin packer? gobuffalo/packr
+    - Jacob: Would be good to embed schemas for their parameters
+        - Radu: We could bundle the schema in thick bundles
+            - Matt: Means we'd be asking bundle if it's valid
+- #285 meeting?
+    - Glyn is going to plan this
```

Comment:
Actually, it's Ralph.

Reply:
cc @squillace

Reply:
https://github.com/cnabio/community/pull/27

Reply:
it was ralph. And as in channel, we should have it be a formal scheduled meeting item, as we'll chew through it all and leave it in bits